manager/state/raft/raft.go

```diff
@@ -476,7 +476,13 @@ func (n *Node) stop() {
 			_ = member.Conn.Close()
 		}
 	}
-	n.Stop()
+
+	if n.Node != nil {
+		n.Stop()
+		n.Node = nil
```

Comment:
Sorry, still confused about this. Now that we set both `n.Node = nil` and `isMember` to 0, isn't `IsStopped` basically the same thing as `IsMember`? I don't understand why we need both. It looks like the only place we use only `IsStopped` is in `LeaderAddr`, but I don't understand why that couldn't use `IsMember` instead.